.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: build-and-test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  checklinks:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check
```

Comment:
Maybe make the name "Check links", like the job title above? Probably not a big deal, I just like a little more descriptive titles when possible.